File: Programming Basics with JS/T04 - Nested Conditional Statements/exercise/02. Summer Outfit.js
function summerOutfit(input) {
    let degrees = Number(input[0]);
    let timeOfTheDay = input[1];
    let outFit ;
    let shoes ;

    if (10 <= degrees && degrees <= 18) {
        if(timeOfTheDay === "Morning") {
            outFit = "Sweatshirt";
            shoes = "Sneakers";
        } else if (timeOfTheDay === "Afternoon" || timeOfTheDay === "Evening") {
            outFit = "Shirt";
            shoes = "Moccasins";
        }

    } else if (18 < degrees && degrees <= 24) {
        if (timeOfTheDay === "Morning" || timeOfTheDay === "Evening") {
            outFit = "Shirt";
            shoes = "Moccasins";
        } else if (timeOfTheDay === "Afternoon") {
            outFit = "T-Shirt";
            shoes = "Sandals";
        }

    } else if (degrees >= 25) {
        if (timeOfTheDay === "Morning") {
            outFit = "T-Shirt";
            shoes = "Sandals";
        } else if (timeOfTheDay === "Afternoon") {
            outFit = "Swim Suit";
            shoes = "Barefoot";
        } else if (timeOfTheDay === "Evening") {
            outFit = "Shirt";
            shoes = "Moccasins";
        }
    }
    console.log(`It's ${degrees} degrees, get your ${outFit} and ${shoes}.`)
}
summerOutfit(["22", "Afternoon"]);

File: Programming Basics with Go/T04 - Conditional Statements (Advanced)/exercise/02. Summer Outfit.go
package main

import "fmt"

func main () {
	var degrees int 
	fmt.Scanln(&degrees)

	var partOfDay string
	fmt.Scanln(&partOfDay)

	var outfit string = "" //дрехите
	var shoes string = ""  //обувки

	if degrees >= 10 && degrees <= 18 {
		//проверка за част от деня -> "Morning", "Afternoon", "Evening"
		switch partOfDay {
			case "Morning":
				outfit = "Sweatshirt"
				shoes = "Sneakers"
			case "Afternoon", "Evening":
				outfit = "Shirt"
				shoes = "Moccasins"
		}
	} else if degrees > 18 && degrees <= 24 {
		//проверка за част от деня -> "Morning", "Afternoon", "Evening"
		switch partOfDay {
			case "Morning", "Evening":
				outfit = "Shirt"
				shoes = "Moccasins"
			case "Afternoon":
				outfit = "T-Shirt"
				shoes = "Sandals"
		}
	} else if degrees >= 25 {
		//проверка за част от деня -> "Morning", "Afternoon", "Evening"
		switch partOfDay {
			case "Morning":
				outfit = "T-Shirt"
				shoes = "Sandals"
			case "Afternoon":
				outfit = "Swim Suit"
				shoes = "Barefoot"
			case "Evening":
				outfit = "Shirt"
				shoes = "Moccasins"
		}

	}

	//знам облекло и обувки
	fmt.Printf("It's %d degrees, get your %s and %s.", degrees, outfit, shoes)

}
